sdk/hardwaresecuritymodules/azure-mgmt-hardwaresecuritymodules/CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Release History
 
+## 1.0.0 (2025-07-02)
+
+### Features Added
+
+  - Enum `ProvisioningState` added member `UPDATING`
+  - Model `CloudHsmClusterBackupStatusOperations` added method `begin_get`
+  - Model `CloudHsmClusterRestoreStatusOperations` added method `begin_get`
+
+### Breaking Changes
+
+  - Deleted or renamed model `CloudHsmClusterResource`
```

Comment:
@ChenxiJiang333 Let us add the migration declaration for first release after migrate to typespec like https://github.com/Azure/azure-sdk-for-python/blob/4e048f89a6e244a530d9d267883403197f7ab78a/sdk/oracledatabase/azure-mgmt-oracledatabase/CHANGELOG.md?plain=1#L88